\documentclass{article}

% Target journal: Molecular Phylogenetics and Evolution
%
% From author guidelines:
%
% Short communications of approximately 3000 words are also accepted. 
% These papers should contain no more than two figures, two tables, and thirty references. 
% A short abstract of fewer than 200 words is acceptable.


% Annotation/feedback commands
\newcommand*\rampal[1]{\textcolor{red}{\textbf{[RSE: #1]}}}
\newcommand*\richel[1]{\textcolor{orange}{\textbf{[RJCB: #1]}}}
\newcommand*\gio[1]{\textcolor{green}{\textbf{[GL: #1]}}}

% Bibliography
\usepackage{natbib}
\bibpunct{(}{)}{;}{a}{}{;}

\usepackage{amssymb}
\usepackage[english]{babel}

\usepackage{bbm}
\usepackage{dsfont}


% Use 'It was found that something is something (Name 1234)' style
\setcitestyle{authoryear,open={},close={}}

% Affiliations
\usepackage{authblk}
\title{The error in Bayesian phylogenetic reconstruction when speciation co-occurs}

\author[1]{Giovanni Laudanno}
\author[1]{Rich\`el J.C. Bilderbeek}
\author[1]{Rampal S. Etienne}
\affil[1]{Groningen Institute for Evolutionary Life Sciences, University of Groningen, Groningen, The Netherlands}

% Use double spacing
\usepackage{setspace}
\doublespacing

\usepackage{pgf}
\usepackage{hyperref}
\usepackage{verbatim}
\usepackage{comment}
  
% Adds numbered lines
\usepackage{lineno}
\linenumbers

\hyphenation{
  BEAST
  Pa-ra-me-ter
  Drum-mond 
  Bayes-ian 
  Mr-Bayes 
  ap-proach-es 
  Rev-Bayes 
  cre-ate
  spe-ci-a-tion-com-ple-tion
  pro-trac-ted
 }

\begin{document}

\maketitle

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{abstract}

  % From 'How to construct a Nature summary paragraph'

  % A short abstract of fewer than 200 words is acceptable.

  % One or two sentences providing a basic
  % introduction to the field,
  % comprehensible to a scientist in any discipline.
  There exist millions of species on Earth,
  all originating from a common ancestor billions
  of years ago.
  The field of phylogenetics uses heritable material
  to determine which species are closest related and what are
  the mathematics that shape speciation.
  
  % Two to three sentences of
  % more detailed background, comprehensible to
  % scientists in related disciplines.
  In Bayesian phylogenetics, a DNA/RNA/protein alignment
  is used to infer a distribution of phylogenies and parameter estimates.
  To do so, we use assumptions that may be biologically unrealistic,
  but may give tolerable errors.
  
  % One sentence clearly stating the general
  % problem being addressed by this particular
  % study.
  Contemporary inference assumes that speciation
  never co-occurs.

  % One sentence summarising the main
  % result (with the words “here we show”
  % their equivalent).
  Here we show the error we make in our inference,
  when nature has varying degrees of co-occuring speciation.

  % Two or three sentences explaining what
  % the main result reveals in direct
  % comparison to what was thought to be the case
  % previously, or how the main result adds to
  % previous knowledge.

  % One or two sentences to put the results into a
  % more general context.


  % Two or three sentences to provide a
  % broader perspective, readily comprehensible
  % to a scientist in any discipline, may be included
  % in the first paragraph if the editor considers that the accessibility of the paper is significantly enhanced
  % by their inclusion. Under these circumstances, the length of the paragraph can be up to 300 words.
  % (The above example is 190 words without the final section, and 250 words with it).



\end{abstract}

{\bf Keywords:} computational biology, evolution, phylogenetics, Bayesian analysis, tree prior
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Introduction}
\begin{itemize}

\item There are many contemporary tools that provide the possibility 
to infer a phylogeny from genetic data (DNA, RNA, proteins). 
A popular Bayesian phylogenetic tool is called BEAST (\cite{beast}) 
and its cousin BEAST2 (\cite{beast2}).

\item BEAST is very flexible, providing the user with the option 
to set up all possible phylogenetic priors (e.g. site/clock/speciation model).

%Current limits in current tools.
\item However, currently available priors can be not suitable 
to analyze some specific datasets. 
With this work we aim to test whether or not 
the implementation of a new prior model 
is beneficial to study a specific kind of diversification process.

\item BEAST2 gives us the possibility to introduce new tree priors 
to infer phylogenies based on different assumptions 
on how the speciation process takes place.

\item One of such speciation processes is the multiple birth hypothesis,
a new model (described below) and thus currently absent in BEAST.

\item The Multiple birth hypothesis can be useful to explain a phenomenon 
that has always puzzled evolutionary biologists: 
what are the drivers of the diversification processes 
for those phylogenies that show an impressive amount of speciation events 
in relatively short times? 
The (constant-rate) birth-death (BD) model embodies the common assumption that 
only a single speciation event can occur at any given time.
The multiple-birth-death (MBD) model 
relaxes this assumption, allowing events in which 
large-scale environmental changes lead to a great number of species 
in relatively short time intervals. 
Such a hypothesis may be a better fit to describe the burst in systems 
like cichlid fish diversification in the 
African Great Lakes: Malawi, Tanganyika and Victoria 
(\cite{janzen2016}, \cite{janzen2017}).

\item However, it may be that current BD tree priors are good enough 
at detecting such events, with a (preferred) lower level of complexity. 
If this is the case one should always be more keen to adopt the simplest model.

\item Here we present our study with the aim of exploring 
when using a more complex MBD tree prior is warranted.
To do so, we simulate phylogenies using the MBD process, with
varying degrees of that process. To be explicit, we define
that degree, $s$, as the number of extinct and extant species created
during a co-occuring speciation event, 
$N_{\mathbb{M}}$, from the total number of extinct and extant species:

\begin{equation}
s = \frac{N_{\mathbb{M}}}{N_{\mathbb{M}} + N_{\mathbb{B}}}
\end{equation}


Here, $N_{\mathbb{B}}$ is the number of extinct and extant species created
during a default single-birth speciation event.

From such phylogenies with different degrees of $s$, we measure the
inference error we make today, would nature follow such a phylogeny.
The inference error we make today is caused by the assumption of a BD process
and by inherent noise in this inference. 

We have the hypothesis $\mathcal{H}_1$ that, for a higher $s$, 
the inference error $e$ will increase:

\begin{equation}
e = f(s)
\end{equation} 

Where $f$ is a monotonously increasing function of unknown shape.

\begin{figure}[!htbp]
  \includegraphics[width=\textwidth]{fig_h_1.png}
  \caption{
    Hypothesis 1: we expect the error made be correlated to the number 
    of species created by the multiple-birth process over the total number 
    of species created
  }
  \label{fig:h_1}
\end{figure}

The MBD process has multiple components, that may cause 
different values of $e$ for identical values of $s$.

One MBD component that may cause 
different values of $e$ for identical values of $s$
is the MBD regime: which can be many modest 
multiple-speciation events, or few intense ones. 
As we have no prior expectations, we have the (null) hypothesis, 
$\mathcal{H}_7$, that the MBD regime has no effect on $e$.

\begin{figure}[!htbp]
  \includegraphics[width=\textwidth]{fig_h_7.png}
  \caption{
    Hypothesis 7: there is no difference in inference errors if the MBD process
    generates many species in many modest MB events or few intense MB events
  }
  \label{fig:h_7}
\end{figure}

Another MBD component that may cause 
different values of $e$ for identical values of $s$
is the effect of extinction. 
As extinctions will hit lineages created by both speciation processes equally,
and we have no additionaly prior expectations,
we have the (null) hypothesis, 
$\mathcal{H}_2$, that extinction has no effect on $e$.

\begin{figure}[!htbp]
  \includegraphics[width=\textwidth]{fig_h_2.png}
  \caption{
    Hypothesis 2: the effect of extinction rates is neutral
  }
  \label{fig:h_2}
\end{figure}

Another MBD component that may cause 
different values of $e$ for identical values of $s$,
is the timing of a multiple-birth event: be it close to the
crown age or close to the present. 
Compared to a late multiple birth event, an early multiple birth event may have a 
longer-lasting effect (as the next speciation event will be later), but it
will create less new species, as there are still fewer taxa.
As we have no prior expectations,
we have the (null) hypothesis, 
$\mathcal{H}_3$, that the timing of multiple-birth events has no effect on $e$.

\begin{figure}[!htbp]
  \includegraphics[width=\textwidth]{fig_h_3.png}
  \caption{
    Hypothesis 3: the timing of a multiple birth event has no effect.
  }
  \label{fig:h_3}
\end{figure}

Another MBD component that may cause 
different values of $e$ for identical values of $s$,
is the number of taxa in a phylogeny,
As there is no diversity dependency in any of the processes
and we have no further prior expectations, we have
the (null) hypothesis $\mathcal{H}_5$, that the number of
taxa has no effect on $e$.
As a higher number of taxa increases the information content in a
phylogeny, we have hypothesis $\mathcal{H}_6$ that the
variance in $e$ decreases.

\begin{figure}[!htbp]
  \includegraphics[width=\textwidth]{fig_h_5.png}
  \caption{
    Hypothesis 5: the number of taxa does not
    have an effect on the error being made
  }
  \label{fig:h_5}
\end{figure}

\begin{figure}[!htbp]
  \includegraphics[width=\textwidth]{fig_h_6.png}
  \caption{
    Hypothesis 6: for a higher number of taxa 
    the variance in the error decreases
  }
  \label{fig:h_6}
\end{figure}

\end{itemize}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Methods}

\subsection{Model}
\begin{itemize}

\item 
\richel{TODO: move to Introduction}
Current phylogenetic tools assume that only a single speciation event 
can occur at any given time.
While this assumption is useful to construct a wide variety of successful 
models (e.g \cite{Maddison2007biSSE}, \cite{Valente2015}, 
\cite{etienne2012diversity}, \cite{etienne2014estimating}),
they disallow for environmental changes that trigger speciations 
in multiple clades at a same point in time. 

\item 
\richel{TODO: move to Introduction}
In the MBD model, parameters $\lambda$ and $\mu$ correspond, respectively, 
to the common per-species speciation and extinction rates present 
also in the standard BD model. 
Additionally, MBD relies on two additional parameters. 
Parameter $\nu$ is the rate at which an environmental change is triggered.
When such event is triggered, 
all species present in the phylogeny at that moment
have a probability $q$ to speciate at that time, which is 
independent on $\lambda$. 
Polytomies are not allowed in such process 
as each species can speciate only once at the time.

\item It is also possible to write down a likelihood function 
for such processes as in \cite{mbd}.
    
\end{itemize}

\subsection{Simulations}
\begin{itemize}

\item To investigate the effect of $s$ on $e$, we simulate phylogenies
for different values of $s$ spread equally from zero (no multiple-birth
event, thus a BD model) to one (species are created only from multiple-birth
events) and three intermediate values of 0.25, 0.5 and 0.75. Aggregating all
components of $s$, we show the effect of $s$ on $e$ in figure \ref{fig:1}.

\item To see the effect of the number of taxa on the relation 
between $s$ and $e$, we simulate phylogenies
for different number of extant taxa $n$. We use values of $n$
of 50, 100 and 200 extant taxa, as this will result in phylogenies
that are big enough to be useful, yet small of enough to
be computationally feasible. For the different values of $n$, we show
the relation between $s$ and $e$ in figure \ref{fig:5} and its effect
on the variance of $e$ in figure \ref{fig:6}.

\item To see the effect of the extinction rate on the relation 
between $s$ and $e$, we simulate phylogenies
with different extinction rates $\mu$. We use values of $\mu$
of 0.0, 0.1 and 0.2. An extinction rate of zero has two features:
(1) the model falls back to a pure multiple-birth model, 
(2) all multiple-birth events are observed.
For the different values of $\mu$, we show
the relation between $s$ and $e$ in figure \ref{fig:2}.

\item We start simulating $N_{S} = 1000$ MBD trees,
with either 50, 100 and 200 taxa.

\item From each MBD tree, a DNA sequence alignment is simulated. 
For each sequence alignment we then perform a Bayesian analysis 
to recover a posterior distribution of trees, 
each composed of $N_{P}$ phylogenies. 
Such analysis is performed using 
the 'pirouette' package (\cite{pirouette}) to call the BEAST2 tool 
suite from R. 
We let the Bayesian analysis assume a BD prior in both cases, 
to investigate the extent of the error we make under this assumption.

\item For each tree generated under the MBD model 
we aim to generate a "twin" tree under the BD model. 
With the word "twin" 
we denote a tree generated starting from the respective MBD tree, 
in order to perform a fair comparison with it. 
This operation has to be done, 
because we want to compare two trees 
that are generated by different processes. 
To do so we infer the parameters $\lambda_{BD}$ and $\mu_{BD}$ 
from the MBD maximizing the likelihood under a BD model. 
To perform this operation we use the function "\texttt{bd\_ML}" 
from the package "\texttt{DDD}" (\cite{etienne2012diversity}). 

\item We then exploit such parameters to generate a BD tree 
using the function "\texttt{tess.sim.taxa.age}" 
from the package "\texttt{TESS}" (\cite{Hoehna2013}). 
We simulate the tree in such a way the new tree 
has the same number of tips and the same crown age as the MBD tree. 
We furthermore require that the BD tree conserve the topology of the MBD tree.
We have hypothesis H4 that, compared to the MBD trees, 
the error will be less in the BD twin tree.
The difference between the errors made in MBD and twin BD trees indicates
the impact the MBD process has on the error we make in inference using a
contemporary BD prior.

\begin{figure}[!htbp]
  \includegraphics[width=\textwidth]{fig_h_4.png}
  \caption{
    Hypothesis 4: compared to the MBD trees, 
    the error will be less in the BD twin tree
  }
  \label{fig:h_4}
\end{figure}


We want the MBD and twin BD trees to contain the same amount of information, 
i.e. the same number of DNA mutations and the same number of taxa at the present:

\begin{equation}
m_{MBD} = m_{BD} \label{m equivalence}
\end{equation} 

The expected number of mutations $m$ of a phylogeny 
with crown age $-T$ (with $T>0$) in fact is given by
\richel{
  So one of use likes '-T', the other likes 'T'. How to resolve this?
}

\begin{equation}
m = L \cdot \rho \cdot \int_{0}^{T} n(t)\ dt \label{m calculation}
\end{equation}

where $L$ is the number of DNA nucleotides, 
$\rho$ is the per-site per-species mutation rate and
$n(t)$ the number of species at each time.

The parameter we'll tune is $\rho$ ... \richel{elaborate here :-)}

Since we cannot know $n_{BD}(t)$ before running simulations
we need to replace it with a proxy. 
For this reason we will use the average number of
species in time according to the BD model. 
It's well known that this is equal to \gio{insert proper citation}

\begin{equation}
    <n_{BD}>(t) = n_{0} \cdot e^{(\mu_{BD} - \lambda_{BD})t} \label{BD average n}
\end{equation}

where $n_{0} = n_{BD}(-T) = n_{MBD}(-T)$ is the initial number of species 
at the crown age.
From \ref{m equivalence}, \ref{m calculation} and \ref{BD average n} follows:

\begin{equation}
m_{MBD} = L \cdot \rho \cdot \int_{0}^{T} <n_{BD}>(t)\ dt \\
= L \cdot \rho \cdot n_{0} \cdot \left[ \frac{e^{(\mu_{BD} - \lambda_{BD})T} - 1}{\mu_{BD} - \lambda_{BD}} \right]
\end{equation}

If we set $\mu_{BD} = \mu_{MBD}$ and reverse this relation 
we can extrapolate the value of $\lambda_{BD}$ to use to generate BD trees.

\begin{figure}[!htbp]
  \includegraphics[width=\textwidth]{mbd.jpg}
  \caption{
    How to create twin trees and alignments. 
    From a focal MBD tree, a twin tree is produced as 
    such: (1) estimate the $\lambda_{BD}$ to get 
    the same expected number of tips, (2) simulate a BD tree 
    with that amount of tips (discard trees with different number of tips), 
    (3) estimate a mutation rate to get an alignment 
    with the same expected number of mutations, (4) simulate alignments 
    with that amount of mutations (discard those that don't, 
    the picture shows an alignment that should be discarded) 
  }
\end{figure}

\item We explained how we set the parameters for each twin BD tree. 
Using this rules we generate a BD dataset. 
We repeat the analysis, producing alignments for each tree 
and subsequently using BEAST to produce a posterior for each of them.

\subsection{Measuring the inference error}

\item So far we have simulated two datasets of trees under the two models: 
$\{T_{i}^{BD}\}_{i=1}^{N_{S}}$ and $\{T_{i}^{MBD}\}_{i=1}^{N_{S}}$.
We used them to generate a dataset of alignments for each model: $\{X^{BD}_{i}\}_{i=1}^{N_{S}}$ and $\{X^{MBD}_{i}\}_{i=1}^{N_{S}}$. From each dataset we produced a posterior distribution from a BD prior: 
$P_{i}(\theta | X^{BD}_{i}, BD)$ and $P_{i}(\theta | X^{MBD}_{i}, BD)$.
\gio{
  1) We might want to rename the models, e.g. BD = (0) and MBD = (1). 
  These names with capital letters are too big and ugly;
}
\richel{
  I would strongly prefer MBD and BD, as I feel replacing the big ugly 
  capital letters by short pretty numbers hurts readability even more 
}

\item To compare the results for the two models we measure the inference 
error using the nLTT statistic between known/true tree and 
posterior/inferred trees (\cite{nltt}). 
To obtain such statistics the procedure is the following:

- From each tree $T_{i,j}^{M}$ (with $j=1,...,N_{S}$) 
  belonging to the posterior $P_{i}(\theta | X^{M}_{i}, BD)$ 
  and relative to the model $M$, we extrapolate the lineage-through-time (LTT), 
  in other words we measure the number of species as a function of 
  time $n_{i,j}(t)$. To allow a comparison we normalize dividing by the 
  maximum number of species of each tree, i.e. the number of tips at the 
  present $N_{i,j}(t)=\frac{n_{i,j}(t)}{n^{max}_{i,j}}$. We then define the 
  nLTT measure as

$nLTT_{i,j} = \int_{0}^{T} | N_{i,j}(t) - N_{T_{i}} | dt$

\gio{I am running out of letters :(}
\richel{Haha! I suggest to use the same equation and symbols 
  as equation 1 in
  the nLTT article of Janzen, Hoehna and Etienne, 2015:
}

$$
\Delta nLTT = \int_{0}^{1} | nLTT_1(t) - nLTT_2{t} | dt
$$

\subsection{Model selection}

We simulate alignments using the simplest nucleotide substitution model (JC69),
the simplest clock model (strict). It is thus imminent to assume these
models in our Bayesian inference. Nevertheless, the phylogeny the alignment
was based on, could have followed either an MBD or BD tree model, 
where we in both cases assume a BD tree model. This will have 
an unknown effect on our inference: it may theoretically be that an MBD model
generates (a tree that generates) an alignment in which a different site 
and/or clock model is favored. 

We investigate this by measuring if the generative model (with the simplest
nucleotide substitution and simplest clock model) is indeed selected 
to be the best fitting model. 
To be precise, we look at the model 
with the highest marginal likelihood 
(also called evidence \cite{mackay2003information}),
$f(D|M)$, which is the probability of the data D given model M.
In the context of this research, D consists of the DNA alignment,
and M is the combination of site, clock and tree models.

To estimate the marginal likelihood, 
we use an algorithm named nested sampling \cite{skilling2006nested}.
Nested sampling is attractive to use
in a phylogentic context, as it gives a good estimation,
requires little tuning \cite{maturana2018}.
Nested sampling is available as a BEAST2 package
and can be used by babette \cite{babette}.

The nested sampling algorithm stops its run 
when the marginal likelihood estimation error 
reaches below a certain tolerance.  
Similar to \cite{maturana2018},
we use a (relative) error tolerance $\epsilon$ of $10^{-13}$,
1 particle to explore the parameter space
and 100 active points. 
To achieve the latter, we use the MCMC chain length $L_c$ of 1M 
(as also used in the parameter estimates),
and a sub-chain length $L_{sc}$ of 10K.

The models we use in our model comparison are the four combinations
of two site models and two clock models. We use the JC69 site model, which
is the (generative and) simplest model and GTR, the site model with most
degrees of freedom. For the clock models, we use the strict clock model,
which is the (generative and) simplest clock model, and the RLN clock model.
\richel{Could also just be all site models and clock models = 8 models}

From these four marginal likelihood estimates, we calculate the weight of
the generative model and plot this in figure 2. We do this for both the 
alignments derived from the MBD tree and the BD twin tree. We expect that
the generative model has the heighest weight in both the MBD and BD alignments.
We expect this weight to be higher in the BD alignments.

\end{itemize}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Results}
\begin{itemize}

\item We expected the effect of $s$ on $e$ to be monotonously increasing
function (figure \ref{fig:h_1}). Figure \ref{fig:1} shows the effect 
measured in this experiment. 

\begin{figure}[!htbp]
  \includegraphics[width=\textwidth]{fig_1.png}
  \caption{
    The effect of s on e
  }
  \label{fig:1}
\end{figure}

\item We expected the effect of the number of taxa on the relation
between $s$ and $e$ to be neutral (figure \ref{fig:h_5}). 
Figure \ref{fig:5} shows the effect 
measured in this experiment. 

\begin{figure}[!htbp]
  \includegraphics[width=\textwidth]{fig_5.png}
  \caption{
    The effect of number of taxa on the relation between s on e
  }
  \label{fig:5}
\end{figure}

\item We expected the effect of the number of taxa on the variance
of $e$ for similar values of $s$ to decrease (figure \ref{fig:h_6}). 
Figure \ref{fig:6} shows the effect 
measured in this experiment. 

\begin{figure}[!htbp]
  \includegraphics[width=\textwidth]{fig_6.png}
  \caption{
    The effect of number of taxa on the variance in e
    for similar values of s
  }
  \label{fig:6}
\end{figure}

\item We expected the effect of extinction on the relation
between $s$ and $e$ to be neutral (figure \ref{fig:h_2}). 
Figure \ref{fig:2} shows the effect 
measured in this experiment. 

\begin{figure}[!htbp]
  \includegraphics[width=\textwidth]{fig_2.png}
  \caption{
    The effect of extinction on the relation between s on e
  }
  \label{fig:2}
\end{figure}


\item

\end{itemize}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Bibliography % MEE style
\bibliographystyle{mee}
\bibliography{article}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\appendix

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{table}
  \centering 
  \begin{tabular}{r c c c}
    \hline
    Symbol                & Expectation & Expected figure & Measured figure \\
    \hline
    \hline
    $\mathcal{H}_1$       & For a higher $s$, $e$ will increase monotonously & \ref{fig:h_1} & \ref{fig:1} \\
    $\mathcal{H}_2$       & Extinction rates have no effect on $e$ & \ref{fig:h_2} & \ref{fig:2} \\
    $\mathcal{H}_3$       & The timing of multiple-birth events has no effect on $e$ & \ref{fig:h_3} & Not yet \\
    $\mathcal{H}_4$       & An MBD tree will have a higher $e$ than its twin BD counterpart & \ref{fig:h_4} & Not yet \\
    $\mathcal{H}_5$       & The number of taxa has no effect on $e$ & \ref{fig:h_5} & \ref{fig:5} \\
    $\mathcal{H}_6$       & A higher number of taxa decreases the variance in $e$ & \ref{fig:h_6} & \ref{fig:6} \\
    $\mathcal{H}_7$       & The MBD regime has no effect on $e$ & \ref{fig:h_7} & Not yet \\
    \hline
  \end{tabular}
  \caption{
    Overview of the symbols. 
  }
  \label{table:hypotheses}
\end{table}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{table}
  \centering 
  \begin{tabular}{r l}
    \hline
                          & Description\\
    \hline
    \hline
    $\lambda$             & Speciation rate from default single-birth process \\
    $\mu$                 & Extinction rate \\
    $\nu$                 & Multiple-birth event trigger rate \\
    $q$                   & Proportion of taxa that speciate during a multiple-birth event \\
    $e$                   & Inference error \\
    $n_{\mathbb{M}}$      & Number of extant taxa created by a multiple-birth event \\
    $n_{\mathbb{B}}$      & Number of extant taxa created by a single-birth event \\
    $N_{\mathbb{M}}$      & Number of extinct and extant taxa created by a multiple-birth event \\
    $N_{\mathbb{B}}$      & Number of extinct and extant taxa created by a single-birth event \\
    $s$                   & Extent/strength of the multiple birth process \\
    $L_c$                 & MCMC chain length \\
    $L_{sc}$              & MCMC sub-chain length \\
    $\epsilon$            & relative error tolerance in marginal likelihood estimation \\
    \hline
  \end{tabular}
  \caption{
    Overview of the symbols. 
  }
  \label{table:symbols}
\end{table}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{table}
  \centering 
  \begin{tabular}{r l}
    \hline
                          & Value(s) \\
    \hline
    \hline
    $s$                   & 0.0, 0.25, 0.5, 0.75, 1.0 \\
    $\mu$                 & 0.0, 0.1, 0.2 [TODO: can be more extreme] \\
    $L_c$                 & $10^6$ \\
    $L_{sc}$              & $10^4$ \\
    $\epsilon$            & $10^{-13}$ \\
    \hline
  \end{tabular}
  \caption{
    Overview of the simulation parameters. 
  }
  \label{table:parameters}
\end{table}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{table}
  \centering 
  \begin{tabular}{ l | c | c | c | c   c   c | c }
    \hline
idx & $n_{taxa}$ & $f_{taxa}^{MB}$ & $\mu$ & MB regime & $n_{\nu}$ evts & $q$ & $\lambda$ \\
    \hline
    \hline
1 & 50 & 0 & 0 & many modest  & 8 & 1 / 8 & derive \\
2 & 50 & 0 & 0 & intermediate & 4 & 1 / 4 & derive \\
3 & 50 & 0 & 0 & few intense & 2 & 1 / 2 & derive \\
4 & 50 & 0 & 0.1 & many modest  & 8 & 1 / 8 & derive \\
5 & 50 & 0 & 0.1 & intermediate & 4 & 1 / 4 & derive \\
6 & 50 & 0 & 0.1 & few intense & 2 & 1 / 2 & derive \\
7 & 50 & 0 & 0.2 & many modest  & 8 & 1 / 8 & derive \\
8 & 50 & 0 & 0.2 & intermediate & 4 & 1 / 4 & derive \\
9 & 50 & 0 & 0.2 & few intense & 2 & 1 / 2 & derive \\
10 & 50 & 0 & 0.2 & many modest  & 8 & 1 / 8 & derive \\
11 & 50 & 0.25 & 0 & intermediate & 4 & 1 / 4 & derive \\
12 & 50 & 0.25 & 0 & few intense & 2 & 1 / 2 & derive \\
13 & 50 & 0.25 & 0 & many modest  & 8 & 1 / 8 & derive \\
14 & 50 & 0.25 & 0.1 & intermediate & 4 & 1 / 4 & derive \\
15 & 50 & 0.25 & 0.1 & few intense & 2 & 1 / 2 & derive \\
16 & 50 & 0.25 & 0.1 & many modest  & 8 & 1 / 8 & derive \\
17 & 50 & 0.25 & 0.2 & intermediate & 4 & 1 / 4 & derive \\
18 & 50 & 0.25 & 0.2 & few intense & 2 & 1 / 2 & derive \\
19 & 50 & 0.25 & 0.2 & many modest  & 8 & 1 / 8 & derive \\
20 & 50 & 0.25 & 0.2 & intermediate & 4 & 1 / 4 & derive \\
21 & 50 & 0.5 & 0 & few intense & 2 & 1 / 2 & derive \\
22 & 50 & 0.5 & 0 & many modest  & 8 & 1 / 8 & derive \\
23 & 50 & 0.5 & 0 & intermediate & 4 & 1 / 4 & derive \\
24 & 50 & 0.5 & 0.1 & few intense & 2 & 1 / 2 & derive \\
25 & 50 & 0.5 & 0.1 & many modest  & 8 & 1 / 8 & derive \\
26 & 50 & 0.5 & 0.1 & intermediate & 4 & 1 / 4 & derive \\
27 & 50 & 0.5 & 0.2 & few intense & 2 & 1 / 2 & derive \\
28 & 50 & 0.5 & 0.2 & many modest  & 8 & 1 / 8 & derive \\
29 & 50 & 0.5 & 0.2 & intermediate & 4 & 1 / 4 & derive \\
30 & 50 & 0.5 & 0.2 & few intense & 2 & 1 / 2 & derive \\
31 & 50 & 0.75 & 0 & many modest  & 8 & 1 / 8 & derive \\
32 & 50 & 0.75 & 0 & intermediate & 4 & 1 / 4 & derive \\
33 & 50 & 0.75 & 0 & few intense & 2 & 1 / 2 & derive \\
34 & 50 & 0.75 & 0.1 & many modest  & 8 & 1 / 8 & derive \\
35 & 50 & 0.75 & 0.1 & intermediate & 4 & 1 / 4 & derive \\
36 & 50 & 0.75 & 0.1 & few intense & 2 & 1 / 2 & derive \\
37 & 50 & 0.75 & 0.2 & many modest  & 8 & 1 / 8 & derive \\
38 & 50 & 0.75 & 0.2 & intermediate & 4 & 1 / 4 & derive \\
39 & 50 & 0.75 & 0.2 & few intense & 2 & 1 / 2 & derive \\
40 & 50 & 0.75 & 0.2 & many modest  & 8 & 1 / 8 & derive \\
41 & 50 & 1 & 0 & intermediate & 4 & 1 / 4 & 0 \\
42 & 50 & 1 & 0 & few intense & 2 & 1 / 2 & 0 \\
43 & 50 & 1 & 0 & many modest  & 8 & 1 / 8 & 0 \\
44 & 50 & 1 & 0.1 & intermediate & 4 & 1 / 4 & 0 \\
45 & 50 & 1 & 0.1 & few intense & 2 & 1 / 2 & 0 \\
46 & 50 & 1 & 0.1 & many modest  & 8 & 1 / 8 & 0 \\
47 & 50 & 1 & 0.2 & intermediate & 4 & 1 / 4 & 0 \\
48 & 50 & 1 & 0.2 & few intense & 2 & 1 / 2 & 0 \\
49 & 50 & 1 & 0.2 & many modest  & 8 & 1 / 8 & 0 \\
50 & 50 & 1 & 0.2 & intermediate & 4 & 1 / 4 & 0 \\
    \hline
  \end{tabular}
  \caption{
    Overview of the MBD parameters 1/3. 
  }
  \label{table:mbd_parameters_1}
\end{table}

\begin{table}
  \centering 
  \begin{tabular}{ l | c | c | c | c   c   c | c }
    \hline
idx & $n_{taxa}$ & $f_{taxa}^{MB}$ & $\mu$ & MB regime & $n_{\nu}$ evts & $q$ & $\lambda$ \\
    \hline
    \hline
51 & 100 & 0 & 0 & few intense & 2 & 1 / 2 & derive \\
52 & 100 & 0 & 0 & many modest  & 8 & 1 / 8 & derive \\
53 & 100 & 0 & 0 & intermediate & 4 & 1 / 4 & derive \\
54 & 100 & 0 & 0.1 & few intense & 2 & 1 / 2 & derive \\
55 & 100 & 0 & 0.1 & many modest  & 8 & 1 / 8 & derive \\
56 & 100 & 0 & 0.1 & intermediate & 4 & 1 / 4 & derive \\
57 & 100 & 0 & 0.2 & few intense & 2 & 1 / 2 & derive \\
58 & 100 & 0 & 0.2 & many modest  & 8 & 1 / 8 & derive \\
59 & 100 & 0 & 0.2 & intermediate & 4 & 1 / 4 & derive \\
60 & 100 & 0 & 0.2 & few intense & 2 & 1 / 2 & derive \\
61 & 100 & 0.25 & 0 & many modest  & 8 & 1 / 8 & derive \\
62 & 100 & 0.25 & 0 & intermediate & 4 & 1 / 4 & derive \\
63 & 100 & 0.25 & 0 & few intense & 2 & 1 / 2 & derive \\
64 & 100 & 0.25 & 0.1 & many modest  & 8 & 1 / 8 & derive \\
65 & 100 & 0.25 & 0.1 & intermediate & 4 & 1 / 4 & derive \\
66 & 100 & 0.25 & 0.1 & few intense & 2 & 1 / 2 & derive \\
67 & 100 & 0.25 & 0.2 & many modest  & 8 & 1 / 8 & derive \\
68 & 100 & 0.25 & 0.2 & intermediate & 4 & 1 / 4 & derive \\
69 & 100 & 0.25 & 0.2 & few intense & 2 & 1 / 2 & derive \\
70 & 100 & 0.25 & 0.2 & many modest  & 8 & 1 / 8 & derive \\
71 & 100 & 0.5 & 0 & intermediate & 4 & 1 / 4 & derive \\
72 & 100 & 0.5 & 0 & few intense & 2 & 1 / 2 & derive \\
73 & 100 & 0.5 & 0 & many modest  & 8 & 1 / 8 & derive \\
74 & 100 & 0.5 & 0.1 & intermediate & 4 & 1 / 4 & derive \\
75 & 100 & 0.5 & 0.1 & few intense & 2 & 1 / 2 & derive \\
76 & 100 & 0.5 & 0.1 & many modest  & 8 & 1 / 8 & derive \\
77 & 100 & 0.5 & 0.2 & intermediate & 4 & 1 / 4 & derive \\
78 & 100 & 0.5 & 0.2 & few intense & 2 & 1 / 2 & derive \\
79 & 100 & 0.5 & 0.2 & many modest  & 8 & 1 / 8 & derive \\
80 & 100 & 0.5 & 0.2 & intermediate & 4 & 1 / 4 & derive \\
81 & 100 & 0.75 & 0 & few intense & 2 & 1 / 2 & derive \\
82 & 100 & 0.75 & 0 & many modest  & 8 & 1 / 8 & derive \\
83 & 100 & 0.75 & 0 & intermediate & 4 & 1 / 4 & derive \\
84 & 100 & 0.75 & 0.1 & few intense & 2 & 1 / 2 & derive \\
85 & 100 & 0.75 & 0.1 & many modest  & 8 & 1 / 8 & derive \\
86 & 100 & 0.75 & 0.1 & intermediate & 4 & 1 / 4 & derive \\
87 & 100 & 0.75 & 0.2 & few intense & 2 & 1 / 2 & derive \\
88 & 100 & 0.75 & 0.2 & many modest  & 8 & 1 / 8 & derive \\
89 & 100 & 0.75 & 0.2 & intermediate & 4 & 1 / 4 & derive \\
90 & 100 & 0.75 & 0.2 & few intense & 2 & 1 / 2 & derive \\
91 & 100 & 1 & 0 & many modest  & 8 & 1 / 8 & 0 \\
92 & 100 & 1 & 0 & intermediate & 4 & 1 / 4 & 0 \\
93 & 100 & 1 & 0 & few intense & 2 & 1 / 2 & 0 \\
94 & 100 & 1 & 0.1 & many modest  & 8 & 1 / 8 & 0 \\
95 & 100 & 1 & 0.1 & intermediate & 4 & 1 / 4 & 0 \\
96 & 100 & 1 & 0.1 & few intense & 2 & 1 / 2 & 0 \\
97 & 100 & 1 & 0.2 & many modest  & 8 & 1 / 8 & 0 \\
98 & 100 & 1 & 0.2 & intermediate & 4 & 1 / 4 & 0 \\
99 & 100 & 1 & 0.2 & few intense & 2 & 1 / 2 & 0 \\
100 & 200 & 1 & 0.2 & many modest  & 8 & 1 / 8 & 0 \\
    \hline
  \end{tabular}
  \caption{
    Overview of the MBD parameters 2/3. 
  }
  \label{table:mbd_parameters_2}
\end{table}

\begin{table}
  \centering
  \begin{tabular}{ l | c | c | c | c   c   c | c }
    \hline
idx & $n_{taxa}$ & $f_{taxa}^{MB}$ & $\mu$ & MB regime & $n_{\nu}$ evts & $q$ & $\lambda$ \\
    \hline
    \hline
101 & 200 & 0 & 0 & intermediate & 4 & 1 / 4 & derive \\
102 & 200 & 0 & 0 & few intense & 2 & 1 / 2 & derive \\
103 & 200 & 0 & 0 & many modest  & 8 & 1 / 8 & derive \\
104 & 200 & 0 & 0.1 & intermediate & 4 & 1 / 4 & derive \\
105 & 200 & 0 & 0.1 & few intense & 2 & 1 / 2 & derive \\
106 & 200 & 0 & 0.1 & many modest  & 8 & 1 / 8 & derive \\
107 & 200 & 0 & 0.2 & intermediate & 4 & 1 / 4 & derive \\
108 & 200 & 0 & 0.2 & few intense & 2 & 1 / 2 & derive \\
109 & 200 & 0 & 0.2 & many modest  & 8 & 1 / 8 & derive \\
110 & 200 & 0 & 0.2 & intermediate & 4 & 1 / 4 & derive \\
111 & 200 & 0.25 & 0 & few intense & 2 & 1 / 2 & derive \\
112 & 200 & 0.25 & 0 & many modest  & 8 & 1 / 8 & derive \\
113 & 200 & 0.25 & 0 & intermediate & 4 & 1 / 4 & derive \\
114 & 200 & 0.25 & 0.1 & few intense & 2 & 1 / 2 & derive \\
115 & 200 & 0.25 & 0.1 & many modest  & 8 & 1 / 8 & derive \\
116 & 200 & 0.25 & 0.1 & intermediate & 4 & 1 / 4 & derive \\
117 & 200 & 0.25 & 0.2 & few intense & 2 & 1 / 2 & derive \\
118 & 200 & 0.25 & 0.2 & many modest  & 8 & 1 / 8 & derive \\
119 & 200 & 0.25 & 0.2 & intermediate & 4 & 1 / 4 & derive \\
120 & 200 & 0.25 & 0.2 & few intense & 2 & 1 / 2 & derive \\
121 & 200 & 0.5 & 0 & many modest  & 8 & 1 / 8 & derive \\
122 & 200 & 0.5 & 0 & intermediate & 4 & 1 / 4 & derive \\
123 & 200 & 0.5 & 0 & few intense & 2 & 1 / 2 & derive \\
124 & 200 & 0.5 & 0.1 & many modest  & 8 & 1 / 8 & derive \\
125 & 200 & 0.5 & 0.1 & intermediate & 4 & 1 / 4 & derive \\
126 & 200 & 0.5 & 0.1 & few intense & 2 & 1 / 2 & derive \\
127 & 200 & 0.5 & 0.2 & many modest  & 8 & 1 / 8 & derive \\
128 & 200 & 0.5 & 0.2 & intermediate & 4 & 1 / 4 & derive \\
129 & 200 & 0.5 & 0.2 & few intense & 2 & 1 / 2 & derive \\
130 & 200 & 0.5 & 0.2 & many modest  & 8 & 1 / 8 & derive \\
131 & 200 & 0.75 & 0 & intermediate & 4 & 1 / 4 & derive \\
132 & 200 & 0.75 & 0 & few intense & 2 & 1 / 2 & derive \\
133 & 200 & 0.75 & 0 & many modest  & 8 & 1 / 8 & derive \\
134 & 200 & 0.75 & 0.1 & intermediate & 4 & 1 / 4 & derive \\
135 & 200 & 0.75 & 0.1 & few intense & 2 & 1 / 2 & derive \\
136 & 200 & 0.75 & 0.1 & many modest  & 8 & 1 / 8 & derive \\
137 & 200 & 0.75 & 0.2 & intermediate & 4 & 1 / 4 & derive \\
138 & 200 & 0.75 & 0.2 & few intense & 2 & 1 / 2 & derive \\
139 & 200 & 0.75 & 0.2 & many modest  & 8 & 1 / 8 & derive \\
140 & 200 & 0.75 & 0.2 & intermediate & 4 & 1 / 4 & derive \\
141 & 200 & 1 & 0 & few intense & 2 & 1 / 2 & 0 \\
142 & 200 & 1 & 0 & many modest  & 8 & 1 / 8 & 0 \\
143 & 200 & 1 & 0 & intermediate & 4 & 1 / 4 & 0 \\
144 & 200 & 1 & 0.1 & few intense & 2 & 1 / 2 & 0 \\
145 & 200 & 1 & 0.1 & many modest  & 8 & 1 / 8 & 0 \\
146 & 200 & 1 & 0.1 & intermediate & 4 & 1 / 4 & 0 \\
147 & 200 & 1 & 0.2 & few intense & 2 & 1 / 2 & 0 \\
148 & 200 & 1 & 0.2 & many modest  & 8 & 1 / 8 & 0 \\
149 & 200 & 1 & 0.2 & intermediate & 4 & 1 / 4 & 0 \\
150 & 200 & 1 & 0.2 & few intense & 2 & 1 / 2 & 0 \\
    \hline
  \end{tabular}
  \caption{
    Overview of the MBD parameters 3/3. 
  }
  \label{table:mbd_parameters_3}
\end{table}



%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


\end{document}