\section{Discussion}

We expected to find a larger inference error 
when increasing the amount of co-occurring speciation events. The reason is simple: the tree prior used assumes that speciation events do not occur simultaneously. The higher the number of co-occurring speciation events, the stronger
the deviation from the species tree prior. However, we did not find evidence for this prediction..

The first reason why this may be the case, is because of the noise
generated by the runs that did not converge. We think that the bimodality
in the error distribution is caused by the converging runs giving a low error, and runs with low ESSes giving higher errors. This can be studied further by Filtering away
the runs with a low ESS. However, this will only remove the second mode, but will not explain the similarity of the two error distributions for true and twin trees.

A second reason might be that our choice of MBD parameter settings resulted
in too few multiple-birth events. We chose low values of $\nu$ to
prevent doing inference on huge trees. For larger trees, one might perhaps find a difference in the error distributions. However, we do remark the frequency of multiple-birth events was often larger than 50\%.

The third reason why this we did not detect a difference in inference errors between true and twin trees may
be due to the choice of measuring our error. We used the nLTT
statistic, which may not be sensitive enough to what we consider to be different trees: trees with aligned speciation events and trees where these are not aligned. It is, however, not straightforward to define a metric that is more sensitive to this difference than nLTT. One could perhaps try a shotgun approach by simply applying many commonly used metrics, but we believe that this is unlikely to yield a different outcome. The nLTT statistic at least picks up the sudden increase in number of lineages present in MBD true trees, but not in BD twin trees, but this difference is apparently small relative to other stochastically arising differences between true/twin and the inferred posterior distribution of trees. 

The MBD model is a BD model that allows for speciation events taking place simultaneously, but has
the same drawbacks as the BD model: the expected number of species increases quickly with time (when net formation of species exceeds extinction) and the model does not take into account that speciation takes time.
The MBD model can, at least in the simulations, easily
be extended to have a diversity-dependent speciation rate (as in the 
DDD model) and/or making speciation take time by adding 
an incipient species state (as in the PBD model).
The idea of this experiment, however, was to measure the impact of species pump dynamics on phylogenetic inference that does not assume this, and the comparison of a BD and MBD model seems therefore appropriate, particularly because for diversity-dependent or protracted single-birth-death models there is no standard species tree prior available which would have complicated our analyses. Furthermore, we do not expect that addition of diversity-dependence or protracted speciation alters the inference on whether speciation events take place simultaneously or not.

For our experiment, we used the default \verb;pirouette; setup:
when simulating a DNA alignment, the simplest (JC69) nucleotide
substitution model and the simplest (strict) clock model are used. 
One could argue these models
are overly simplistic and biologically irrelevant. We argue that this
actually improves the clarity of our results for two reasons. The first
reason is a practical one: because the computational load was lowered,
we were able to perform more replicates. For a process with this high
amount of stochasticity, we think this is highly preferable. The second reason
is that we think that this reduces the noise of our results, without 
hurting the experiment: in our setup, it is essential
to assume the same site and clock model in inference as the one that is actually used in generating the data.

When simulating our phylogenies, we used an MBD model with and without
extinction. In the case in which extinction is present,
picking the BD tree prior as the generative tree prior seems justified:
it is the best-fitting standard tree prior. For the MBD model without
extinction, we could have picked the Yule tree prior. We believe, however,
that this is of minor importance.

For the four MBD parameters $\lambda$, $\mu$, $\nu$ and $q$,
we investigated 1, 2, 4 and 3 different values respectively.
We chose to use only one value for $\lambda$, because the proportion of multiple-birth events depends on the ratio
between $\lambda$ and a combination of $\nu$ and $q$.  
The crown age we selected for our experiments was based on trial and error
of pilot experiments. Because the number of species in an MBD process is
expected to increase exponentially, increasing the crown age has a
profound effect on the number of taxa, with the consequence of having
runs that took days to calculate. We decided to priortize the number of replicates at the cost of a lower number of taxa.

Ideally, we would have liked all our ESSes to be above the recommended value of 
200. We are aware ESSes can be increased easily by
making the MCMC chain longer. However, the bimodal
distribution of ESSes is disadvantageous: to reduce the percentage 
of $ESS < 200$ by fifty percent,
we would have had to increase the MCMC chain lengths by a factor of forty.
We preferred to invest this run-time in doing more replicates.

This research measures the impact that the use of a non-MBD tree prior 
has on the inference error in phylogenetic construction for species that are subject to species pump dynamics.
We found that the trees constructed with the standard BD species tree prior are similar (as measured by nLTT) to the true tree. This may be considered to be good news, as the implementation of an MBD species tree prior does not seem necessary. However, our results also suggest that it will be difficult to determine the parameter of the MBD process when one wants to fit the model to data of a system that is known to be subject to species pump dynamics. One way to test this would be to try  birth-death skyline species tree priors and see if they pick up a signal of elevated speciation rates during co-occurring speciation events. These models may be prone to overparametrization because they have to assume a speciation rate for each interval in which the multiple speciation events take place, whereas the MBD model provides a dynamic explanation for these elevated speciation rates. But if they do pick up a signal, then it may still be worth implementing a species tree prior for the MBD model in Bayesian phylogenetic reconstruction tools. 


