\section{Discussion}

We expected to find a bigger inference error 
when increasing the amount of co-occurring speciation events.
The reason is simple: the tree prior used assumes speciation does not
co-occur. The stronger the amount of co-occurring speciation, the stronger
the deviation from the species tree prior. That we did not find this, 
is unexpected.

The first reason why this may be the case, is because of the noise
generated by the runs that did not converge. We expect that the bimodality
in the error distribution is caused by most runs converging (at the lower
errors) and runs with low ESSes (at the higher errors). Filtering away
the runs with a low ESS may help to get a better idea if the expected
inference error still cannot be detected.

The second reason why this we did not detect an inference error may
be due to the choice of our error measuring method: by using the nLTT
statistic. Another method may be better at detecting an inference error.

A third reason might be that the MBD parameter settings chosen, resulted
in too few multiple-birth events. We chose low values of $\nu$ to
prevent doing inference on huge trees. Being able to work on bigger
trees might resolve this problem.

The MBD model is a BD model that allows for co-occurring speciation,
with the same drawbacks as the BD model: it allows for an unlimited 
number of species and does not take into account that speciation takes time.
That does make the MBD model a very focussed model, only taking into 
account the essence of co-occurring speciation. The MBD model can probably
be extended to have a diversity-dependent speciation rate (as in the 
DDD model) and/or making speciation take time by adding 
an incipient species state (as in the PBD model).
The idea of this experiment, however, is to measure the impact a novel
tree prior has on the inference error. To do so, we could have picked
any other non-standard (as in, not included within BEAST2) tree prior.

For our experiment, we used the default \verb;pirouette; setup:
when simulating a DNA alignment, the simplest (JC69) nucleotide
substitution model and the simplest (strict) clock model are used. 
One could argue these models
are overly simplistic and biologically irrelevant. We argue that this
actually improves the clarity of our results for two reasons. The first
reason is a practical one: because the computational load was lowered,
we were able to perform more replicates. For a process with this high
amount of stochasticity, we think this is highly favored. The second reason
is that we think this reduces the noise of our results, without 
hurting the experiment: in our setup, it is essential
to assume the same generative site and clock model as the one that is actually 
used.

When simulating our phylogenies, we used an MBD model with and without
extinction. In the case in which extinction is present,
picking the BD tree prior as the generative tree prior seems justified:
it is the best-fitting standard tree prior. For the MBD model without
extinction, we could have picked the Yule tree prior. We expect, however,
that this is of minor relevance.

\subsection{Choices to constrain time}

From the four MBD parameters $\lambda$, $\mu$, $\nu$ and $q$,
we investigated 1, 2, 4 and 3 different values respectively.
We chose to use only one $\lambda$, as the proportion of species
created in a co-occurent speciation event is dependent on the ratio
between $\lambda$ and a combination of $\nu$ and $q$.  

The crown age we selected for our experiments was based on trial and error
of pilot experiments. Because the number of species in an MBD process is
expected to increase exponentially, increasing the crown age has a
profound effect on the number of taxa, with the consequence of having
runs that took days to calculate. We decided to prefer a lower number of
taxa at the gain of having more replicates.

Ideally, our ESSes would all be above the recommended value of 
200. We are aware ESSes can be increased easily by
making the MCMC chain longer. However, the bimodal
distribution of ESSes is disadvantageous: to reduce the percentage 
of $ESS < 200$ by fifty percent,
we would have to increase the MCMC chain lengths by a factor of forty.
We invested this run-time in doing more replicates.

\subsection{Future research}

This research measures the impact an MBD tree prior 
has on the inference error.

Because the MBD tree prior has only recently been 
developed, it's biological relevance is unknown.
Would the MBD tree prior be employed on biological data of a system
in which co-occuring speciation is known to have taken a strong role,
it will be hard to conclude its performance is superior to a
rate-shift model or an early-burst model. One way to do so, is
to add the MBD other speciation models as a species tree prior
in a phylogentic tool and do a model comparision (e.g. using nested sampling).
Another way to do so, is measure the impact a rate-shift or early-burst model
has on the inference error in the same way as this research.
Doing so would also allow to put the quantitative results into perspective,
helping us in how to judge the error distributions. 


