\section{Methods}

\subsection{Simulation model}

The multiple-birth-death (MBD) model inherits the parameters $\lambda$ and $\mu$ from the BD model; they correspond, respectively, 
to the traditional per-species speciation and extinction rates. 
Additionally, the MBD model assumes that external events, occuring at rate $\nu$ triggers a speciation initiation event in each lineage which leads to a full new species with probability $q$. 
Whereas parameter $\lambda$ can be interpreted as the rate of sympatric speciation, $\nu$ is the rate of appearance of 
geographical barriers able to interrupt the gene flow in the population,
resulting in a possible allopatric speciation for each of the species.  
Even though multiple speciation events can occur simultaneouly, it does not lead to  
polytomies, because each species can only split once after a trigger event. This model can be easily simulated with a Doob-Gillespie algorithm. A probability distribution for the phylogeny under the MBD model can also be formulated using the integration approach developed for diversity-dependent diversification models \citep{etienne2012diversity}. While this probability distribution could in principle be used as a tree prior in Bayesian phylogenetic inference, it is computationally very demanding, particularly for large trees. With the MBD model we generate simulated datasets for various parameter settings, using the \verb;mbd_sim; function from the \verb;mbd; R package [\citep{mbd}].

\subsection{Estimating the inference error}

From each simulated 'true' MBD tree, we measure the impact of
ignoring the more complex and non-standard MBD tree prior in
Bayesian phylogenetic inference with the R package \verb;pirouette; [\citep{pirouette}].

\verb;pirouette; starts from a 'true'  phylogeny (in our case: the simulated MBD tree), and simulates a DNA sequence alignment on it using a known nucleotide substitution model and a clock model. 
From each sequence alignment, a Bayesian inference is run with a particular choice of tree prior and substitution and clock models. One can choose the same substitution and clock models as used in generating the tree, or pick the ones that fit the data best. For the tree prior we assume the BD model (as the effect of this choice is our focus). We obtain a posterior distribution of jointly-estimated trees and model parameter estimates.
By comparing the true tree and the posterior trees, an inference error distribution is generated. For this comparison we used the absolute nLTT statistic by \citet{janzen2015}, which results in an error distribution with values ranging from zero (when the inferred tree is identical to the true tree) to a maximum of one (trees are completely different).
Another advantage of using the nLTT statistic is that its behavior is best
explored in \citet{bilderbeek2019pirouette}.

We used the twinning option available in \verb;pirouette; that allows to quantify the impact of assuming a wrong tree prior in a Bayesian inference compared to a reference background error that would arise even if the models used in inference were identical to those used in generating the tree (i.e. the twin tree was generated with a BD model).
If the error distribution of the true tree matches the error distribution
of the twin tree, the effect of using an incorrect tree prior is negligible \citet{bilderbeek2019pirouette}.

\subsection{Parameter settings}

We ran multiple pilot experiments with increasingly more replicates to arrive 
at our final parameter settings. We devised a set of rules to make a 
verdict about the settings.  

\begin{itemize}
  \item quality: 95\% of all individual runs should have an effective
    sample size (ESS) of at least 200, as is recommended by \cite{beastbook}.
  \item feasibility: 95\% of all individual runs should finish within 10 days.
  \item reproducibility: the mean run-time of all finished runs should be less than 24 hours 
  \item relevance 1: the percentage of taxa created by the MB process should be
    as high as possible
  \item relevance 2: the percentage of taxa should be
    as high as possible
\end{itemize}

We searched through parameter space until these criteria were met.
This resulted in the simulation parameters in Table~\ref{tab:simulation_parameters}.

\input{tab_simparameters.tex}

We generated alignments that are 1000 nucleotides in length, with a known root sequence of four 250 mono-nucleotide blocks, generated using the simplest nucleotide substitution model (Jukes Cantor, JC69) and clock model (strict), with a mutation rate of $\frac{0.5}{t_c}$, where $t_c$ is the crown age. 
With this mutation rate, each nucleotide has a 50\% chance to mutate (both silently and non-silently) from the ancestral root sequence to any of the contemporary species' sequences at the tips. 

For the Bayesian inference, we assumed a generative model of a site model that
follows a JC69 nucleotide substitution model, a strict clock model and a BD tree prior.
Additionally, we used a Most Recent Common Ancestor (MRCA) prior equal to the crown age with a normal distribution of width $\sigma = 0.01$. We used a Markov Chain Monte Carlo (MCMC) setup of $10^7$ states with a sampling interval of once per $10^4$ states. Of the resulting $10^3$ states, we discarded a burn-in of $10\%$.

For each of the MBD parameter settings, we simulated 40 different trees
that we put through the same \verb;pirouette; pipeline as described above.
Of each of these 40 replicates, we aggregated the error distributions.



