\section{Introduction}

Modern computational techniques, such as BEAST (\citep{beast,beast2}), RevBayes (\citep{hohna2016revbayes}) and MrBayes (\citep{huelsenbeck2001mrbayes,ronquist2003mrbayes}), allow to infer phylogenetic trees from genetic data 
such as DNA, RNA or proteins.
They return posterior distributions of phylogenies 
and estimated parameters by running a Bayesian analysis, 
given aligned sequence data and a set of models. 
One of these models is the diversification model, for which a prior distribution must be provided. Within the Bayesian framework this is called a tree prior; it is a mathematical description 
of the probability distribution of possible branching patterns before looking at the data. Together with the signal from the data, this tree prior will determine the posterior distribution of phylogenies, i.e. after considering the data. Other models include the nucleotide substitution model (i.e. a model of relative transition rates between different nucleotides through time) and the clock model (a model determining the absolute rate of changes for each lineage). For each of these models choices must be made and prior distributions must be specified for their parameters. BEAST2 gives the user the option to set up 
several possible phylogenetic 
priors (e.g. substitution/clock/diversification models). 
However, currently available priors 
might be not suitable to analyze some specific datasets.
For this reason BEAST2 provides users with the possibility 
to introduce new models and corresponding priors. Particularly, one can specify the tree prior for a new model of diversification.

Current phylogenetic tools such as BEAST2 assume that 
only a single speciation event can occur at any given time.
This assumption is consistent with many different diversification models (e.g \cite{Maddison2007biSSE}, \cite{Valente2015}, 
\cite{etienne2012diversity}, \cite{etienne2014estimating}). However, multiple speciation events can take place simultaneously and repeatedly when populations are intermittently disconnected and connected, for example due to climatic fluctuations. This has been called the species pump hypothesis (\citep{haffer1969speciation}) and has been invoked particularly in mountainous areas that underwent glaciation (\citep{muellner2019origins}). Our own interest in the species pump hypotheses arose from its potential explanation of the radiation of cichlid fish in the African rift lakes (Malawi, Tanganyika and Victoria), where water level drops created multiple smaller lakes providing the opportunity for allopatric speciation in multiple species.  (\citep{verheyen1996mitochondrial}, \citep{sturmbauer2001lake}, \citep{janzen2017}).

One could study whether the species pump hypothesis is a viable explanation in empirical ystems by comparing divergence times of sister taxa (\citep{oaks2019comparative}). A more inclusive approach would involve using a model allowing multiple simultaneous speciation events as a new species tree prior in phylogenetic reconstruction. However, introducing a new tree prior may be computationally prohibitive (\citep{bilderbeek2019pirouette}), and may also not be necessary, as current standard birth death (BD) tree priors might prove to be good enough at inferring the correct tree. Here we use the R package \verb;pirouette; (\citep{pirouette}) to check whether this is the case by simulating phylogenies under a species pump model, i.e. the multiple-birth-death model (MBD), with the \verb;mbd; package (\citep{mbd}), then simulating sequence alignments for each of these trees and finally inferring a phylogenetic tree using BEAST2 from these alignment. By comparing the inferred phylogeny with the true (simulated) one, we measure the inference error made by adopting a standard BD tree prior.